rts/System/Platform/CpuID.h

```diff
@@ -44,6 +45,7 @@ namespace springproc {
 		    cores. */
 		int GetNumPhysicalCores() const { return numPhysicalCores; }
 		int GetNumLogicalCores() const { return numLogicalCores; }
+		std::string GetCPUBrand() const { return cpuid_brand; }
```

Comment:
1) looks unrelated to OpenGL driver debug messages (it looks ok so I don't mind that much, but maybe it's accidental part of some larger WIP?)
2) perhaps would be good if it was a const reference? It doesn't look like something that would need copying by default and it's in a singleton so there's no worry about lifetime.